### PR TITLE
task: Adding maintenance operator charts

### PR DESCRIPTION
**Pull request by heyvister1**
Adding maintenance operator charts

### CODE REVIEW

File: deployment/network-operator/Chart.yaml [hunk 1]
```diff
@@ -24,3 +24,7 @@ dependencies:
   name: nic-configuration-operator-chart
   repository: ''
   version: 0.0.1
+- condition: maintenanceOperator.enabled
+  name: maintenance-operator-chart
+  repository: ''
+  version: 0.1.1
```
Comment (rollandf) [inline deployment/network-operator/Chart.yaml:30]: should be: "0.0.1"

File: deployment/network-operator/values.yaml [hunk 2]
```diff
@@ -181,6 +185,29 @@ nic-configuration-operator-chart:
       name: nic-configuration-operator-daemon
       tag: v0.1.14
 
+# Maintenance Operator chart related values.
+maintenance-operator-chart:
+  operator:
+    image:
+      repository: gghcr.io/mellanox
```
Comment (almaslennikov) [inline deployment/network-operator/values.yaml:192]: typo here, use make release-build to propagate the changes from hack/release.yaml

**Review by almaslennikov — CHANGES_REQUESTED** [review]

File: deployment/network-operator/values.yaml (hunk 2 above)
Comment (heyvister1) [reply deployment/network-operator/values.yaml]: Done

**Review by rollandf — APPROVED** [review]

**Review by almaslennikov — APPROVED** [review]

**Comment by almaslennikov** [comment]
@rollandf do we merge with the documentation CI failing? Looks like some token is not set up there

**Comment by heyvister1** [comment]
> @rollandf do we merge with the documentation CI failing? Looks like some token is not set up there

Its pending a fix which I have introduced
https://github.com/Mellanox/network-operator-docs/pull/134
PTAL

**Pull request merged**